### DISCUSSION TITLE
Giscus no load comment in production site

### DISCUSSION THREAD

**Discussion by duartecancela** [timlrx/tailwind-nextjs-starter-blog#495] [Q&A] [answered]
I have one question, on my local project i can publish and load comments on blog posts, but on production site hosted on Vercel i can publish comments but they are not loading...

Give-me that error on console:

[giscus] Discussion not found. A new discussion will be created if a comment/reaction is submitted.

**Comment by dlarroder** [comment, marked as answer]
This is not an error but a warning. It means that there are still no comments on your post. Once a user comments on your post, a discussion will be created on your repository:

for example -- https://github.com/duartecancela/nextjs-digitaldeveloper/discussions/16 when you comment on [this](https://nextjs-digitaldeveloper.vercel.app/blog/test-post) post.

For the comments not loading in production, have you added your `env` variables to vercel?

![Screen Shot 2022-06-29 at 9 01 51 PM](https://user-images.githubusercontent.com/52998821/176442616-5ec6ba37-aafe-48e5-987d-cf727afed91c.png)

Useful links -- https://vercel.com/docs/concepts/projects/environment-variables

Hope this helps!
